docs/developer-docs/web-apps/browser-js/juno.mdx

```diff
@@ -0,0 +1,31 @@
+# Juno
+
+## Overview
+
+[Juno](https://juno.build/) is a community project that provides a blockchain-as-a-service platform designed for developers to deploy dapps without any blockchain knowledge. It features plug-and-play integration for services such as authentication, hosting, and storage. Juno is framework-agnostic, supporting integration with several different frontend frameworks.
+
+Juno provides a way to deploy dapps onto ICP without using the traditional ICP development workflow that utilizes [`dfx`](/docs/current/developer-docs/getting-started/install/) to create, manage, and deploy canisters.
+
+## Using Juno
+
+To get started with Juno, you can create your first [satellite](https://juno.build/docs/add-juno-to-an-app/create-a-satellite), a 100% on-chain smart contract container that provides secure data storage, authentication, hosting, and also supports custom domains. Juno satellite deployment and management are done through a web-based [console](https://console.juno.build/).
+
+To integrate frameworks and packages with Juno, the [Juno SDK core library](https://www.npmjs.com/package/@junobuild/core) is used. This library can be downloaded using [npm](https://www.npmjs.com/), then used in your local environment to integrate Juno with frameworks such as Vue, Angular, Rust, or React.
```

Comment:
To integrate frameworks and packages into your applications and start building, follow one of the quick-start [guides and tutorials](https://juno.build/docs/category/guides-and-examples). Juno is agnostic, allowing for seamless integration with your favorite frontend technologies, whether you're using Next.js, React, SvelteKit, Vue, Angular, Astro, or any other framework.

Furthermore, if you're aiming to extend the capabilities, you can enhance the features with serverless behaviors running within smart contracts using Rust.